.reek.yml

```diff
@@ -0,0 +1,128 @@
+detectors:
+  Attribute:
+    enabled: false
+    exclude: []
+  BooleanParameter:
+    enabled: true
+    exclude: []
+  ClassVariable:
+    enabled: false
+    exclude: []
+  ControlParameter:
+    enabled: true
+    exclude: []
+  DataClump:
+    enabled: true
+    exclude: []
+    max_copies: 2
+    min_clump_size: 2
+  DuplicateMethodCall:
+    enabled: true
+    exclude: []
+    max_calls: 1
+    allow_calls: []
+  FeatureEnvy:
+    enabled: true
+    exclude: []
+  InstanceVariableAssumption:
+    enabled: false
+  IrresponsibleModule:
+    enabled: false
+    exclude: []
+  LongParameterList:
+    enabled: true
+    exclude: []
+    max_params: 4
+    overrides:
+      initialize:
+        max_params: 5
+  LongYieldList:
+    enabled: true
+    exclude: []
+    max_params: 3
+  ManualDispatch:
+    enabled: true
+    exclude: []
+  MissingSafeMethod:
+    enabled: false
+    exclude: []
+  ModuleInitialize:
+    enabled: true
+    exclude: []
+  NestedIterators:
+    enabled: true
+    exclude: []
+    max_allowed_nesting: 2
+    ignore_iterators: []
+  NilCheck:
+    enabled: false
```

Comment:
Any chance we can enable this one? We see a lot of nil checks :thinking:

Reply:
It makes sense in theory, but the rule is just too aggressive. For example, in `ActAsApiRequest` it was complaining about this line: `request_content_type&.match?(/json/)`

Basically it will complain if we use safe navigation or `try`.

Reply:
Okay, yeah.. I really don't like the `try`, but I think the `&.` is not that bad and we should allow it. Thanks for the clarification.